pkg/cmd/root.go

```diff
@@ -40,7 +40,6 @@ var rootCmd = &cobra.Command{
 		configuration.LoadConfig()
```

Comment:
I am guessing now users will need to provide the `--silent` flag too so their flows don't break?

Reply:
Sorry, I don't follow

Reply:
Based on this commit. We now print debug messaging when using `--plain` which will break a users our flow. For example we fetch the ngrok domain using `--plain`. If a upgrade Doppler warning was printed that would break our ngrok flow. To fix this I am guessing we need to use the `--silent` flag.

Reply:
Gotcha, FYI GitHub doesn't show me what commit you commented on. I didn't fully consider the implication of this change, but I suppose it is a breaking change. I don't think this tiny change is worth another major version bump so I'll remove this. I still think this is a good idea long term to help standardize flag behavior.

Reply:
I'm gonna approve this PR assuming this commit gets dropped.

Reply:
Changes pushed with a different commit